app/src/main/java/com/example/photostomusic/ColorFormActivity.java

```diff
@@ -61,24 +72,42 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // Connect visual and logic elements
         btnConfirmColors = findViewById(R.id.btnConfirmColors);
-        redSpinner = findViewById(R.id.redSpinner);
-        violetSpinner = findViewById(R.id.violetSpinner);
-        blueSpinner = findViewById(R.id.blueSpinner);
-        greenSpinner = findViewById(R.id.greenSpinner);
-        yellowSpinner = findViewById(R.id.yellowSpinner);
-        orangeSpinner = findViewById(R.id.orangeSpinner);
+        spinner1 = findViewById(R.id.colorSpinner1);
+        spinner2 = findViewById(R.id.colorSpinner2);
+        spinner3 = findViewById(R.id.colorSpinner3);
+        spinner4 = findViewById(R.id.colorSpinner5);
+        spinner5 = findViewById(R.id.colorSpinner6);
+        spinner6 = findViewById(R.id.colorSpinner7);
+        spinner7 = findViewById(R.id.colorSpinner8);
+        spinner8 = findViewById(R.id.colorSpinner4);
+
+        // Get the name of hex of each color resource as a string to fill the map
+        // The colors are recovered as an int, then logical AND operation to remove
+        // the transparency of the color, convert this to a HEX string
```

Comment:
nit: define `0x00ffffff` as a variable. also the comment does an ok job, but i'd mention that android uses ARGB or something

Reply:
Got it, will change comment and variable on next commit